src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc

```diff
@@ -71,6 +72,9 @@ typedef SharedPtr<AlertManeuverRequest> CommandPtr;
 class AlertManeuverRequestTest
     : public CommandRequestTest<CommandsTestMocks::kIsNice> {
  public:
+  AlertManeuverRequestTest()
+      : mock_message_helper_(*MockMessageHelper::message_helper_mock()) {}
```

Comment:
@dev-gh Are you sure that expectations for message helper mock are cleared after other unit tests? They can be still expected as this object is static